test/integration/requests.test.ts

```
@@ -788,7 +812,7 @@
     });
 
     it("should not create a new task request if task id is not present in the request body", function (done) {
-      let taskRequestObj = validTaskAssignmentRequest
+      let taskRequestObj = validTaskAssignmentRequest;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'taskRequestObj' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)